src/pages/signup/terms.tsx

```diff
@@ -1,22 +1,32 @@
 import React from 'react';
 import { useState } from 'react';
-//import { useNavigate } from 'react-router-dom';
+import { useNavigate } from 'react-router-dom';
+import { ROUTES } from 'constant';
 
 export default function SignupTerms() {
-   const [agreeCheck, setAgreeCheck] = useState([false, false]);
-   //const navigate = useNavigate();
+   const [agreeCheck, setAgreeCheck] = useState([false, false, false]);
+   const navigate = useNavigate();
 
    const handleCheckboxChange = (index: number) => {
```

Comment:
이벤트를 다루는 함수가 아니므로 on- 의 이름을 쓰면 좋을 것 같네요
```suggestion
   const onChangeCheckbox = (index: number) => {
```

Reply:
네 적용하겠습니다~!